src/rooms/dtos/requests/create-room.dto.ts

```diff
@@ -33,10 +48,19 @@ export class CreateRoomDto {
     return value;
   })
   @IsBoolean()
+  @ApiProperty({
+    description: '비공개 여부',
+    example: true,
+  })
   isPrivate: boolean;
 
   @IsOptional()
   @IsNumberString({ no_symbols: true })
   @Length(4, 4)
+  @ApiProperty({
+    description: '비공개 방 비밀번호 (4자리 숫자)',
+    example: '1234',
+    required: false,
+  })
```

Comment:
문서 스펙을 통해 4자리만 허용함을 알려주고자한다면 아래처럼 표현할 수도 있을거같아요

```suggestion
  @ApiProperty({
    description: '비공개 방 비밀번호 (4자리 숫자)',
    example: '1234',
    required: false,
    minLength: 4,
    maxLength: 4,
  })
```